extra_requirements/lm_benchmark.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lm": [
+    "torch==1.3.0",
+    "tqdm"
```

Comment:
add a reasonable version number (you can use the same as for the flower cnn benchmark)